README.md

```diff
@@ -277,6 +275,23 @@ The `verify` option, if supplied, is called as `verify(req, res, buf, encoding)`
 where `buf` is a `Buffer` of the raw request body and `encoding` is the
 encoding of the request. The parsing can be aborted by throwing an error.
 
+##### defaultCharset
+
+The default charset to parse as, if not specified in content-type. Must be
+either `utf-8` or `iso-8859-1`.
```

Comment:
this allows changing the charset, which is a nice semver-minor; the default should probably be utf-8

Reply:
It is. Documented that in 44a6aa944c05cd865cb2479e2e2d00c816c02c76 now.